Makefile

```diff
@@ -148,19 +148,19 @@ thirdparty-fast:
 	    if [ "$$new_zetasql_version" != "$(ZETASQL_VERSION)" ] ; then \
 		echo "[deps]: thirdparty up-to-date. reinstall zetasql from $(ZETASQL_VERSION) to $$new_zetasql_version"; \
 		$(MAKE) thirdparty-configure; \
-		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) --target zetasql; \
+		$(CMAKE_PRG) --build $(THIRD_PARTY_BUILD_DIR) -j $(NPROC) --target zetasql; \
```

Comment:
does NPROC improve speed ?  this is the parallelism of build all thirdparty deps, parallelism of each dep is handled inside each `cmake/Fetch*.cmake`

Reply:
Yes, speed faster. I can't find -j or other options in Fetch*.cmake, where?

Reply:
zetasql uses bazel, which has its own parallelism strategy  builtin, no need to specify NPROC ?

Reply:
I can’t recall about zetasql, I'll revert this change, and check it when I build the old stdc++ capable image.